src/internal_events/mod.rs

```diff
@@ -1,10 +1,66 @@
 use std::borrow::Cow;
 
+macro_rules! define_events_processed {
+    ($ty:ident, $kind:expr, $name:expr) => {
+        pub(crate) struct $ty;
+
+        impl $crate::internal_events::InternalEvent for $ty {
+            fn emit_logs(&self) {
+                trace!(message = "Processed one event.");
```

Comment:
Hmmm is this something we want?

Reply:
Many (most?) of the pre-macro sources included a trace message like that, so I standardized it in the macro.

Reply:
Works for me!